### ISSUE TITLE
Push notification is not received when the app is closed IOS 10

### ISSUE THREAD

**Issue by Leomhl** [fechanique/cordova-plugin-fcm#119]
Push notification is not received when the app is closed (IOS 10). When de app is in background, when it go to the foreground mode, I receive an alert with the data of my push. But... When the app is closed I don't receive nothing. (Sorry, I don't speak english very well =X)

**Comment by jorchg** [comment]
The issue opened just before yours is this 👎 

https://github.com/fechanique/cordova-plugin-fcm/issues/118

I submitted a pull request, but the author does not seem to be very active on this project.

**Comment by Fayozjon** [comment]
SAME ISSUE, ANY FIXES ????

**Comment by carauzs** [comment]
+1

**Comment by iamntg** [comment]
Any fixes for this issue? please reply

**Comment by jorchg** [comment]
@iamntg It is fixed :/

**Comment by nicejwjin** [comment]
@jorchg was this fixed?? 
I got stuck on same issue, get push on foreground but not in background on notification bar.
Does anyone knows the solution??

**Comment by jorchg** [comment]
@nicejwjin Yeah it was fixed!! I'm using it right now with ios 10.1 and is working fine. I think the Remote background pushes capability must be added in Xcode for this plugin to work correctly.
I always activate push notifications, and remote background pushes and fetch, just in case.

Did you?

**Comment by nicejwjin** [comment]
@jorchg That sounds greate! btw I've got back to iOS dev so I got confused what's Remote background pushes capability in Xcode. Could you give me some instruction page?? I can't find-

**Comment by nicejwjin** [comment]
I've read about remote notifications and I think that would be needed for silence update or just like that. But what I need is just a simple Notification, such as normal previous APNS messages which is displayed on the status bar when the app was killed. Are we on the boat @jorchg ?

**Comment by nicejwjin** [comment]
@jorchg Self answer for most old dumb such like me ;)
https://www.dropbox.com/s/ty3405mmcooff3t/Screenshot%202016-11-25%2021.04.22.png?dl=0

You should just turn on the Capability option on xCode. That's all.

We all miss important things because it's simple..  cheers.

**Comment by tottiCrc** [comment]
I have done everything that says:
  A. Capabilities options:
      1. Push notifications on
      2. Background Modes 
          2.1 Background Fetch
          2.2 Remote Notifications
But the notification doesn't work in background mode, in foreground it works.

* I'm using developer certificate 

Sorry about my english.

**Comment by tottiCrc** [comment]
Fixed, bad implementation in server side.

**Comment by jorchg** [comment]
Glad everything is working now! This should be closed

**Comment by Selvam75** [comment]
Could you please post the implementation in server side?

**Comment by Selvam75** [comment]
I could not receive the notification once the app gets killed in ios only,that's why asked.Please help me quick to fix.

**Comment by tottiCrc** [comment]
Hi, sure I build the server side implementation with node js , using this
package https://www.npmjs.com/package/fcm-node, 2 really important things
to consider are :
1. You need to add priority attribute and high as a value, and MUST be a
sibling of "to" and "notification" attributes (
http://stackoverflow.com/questions/38559107/fcm-notifications-not-displaying-when-app-is-terminated-ios
)
2. The body attribute must be present this will be displayed in the
notification bar in IOS.

If you have problems after this tell me.

Sorry about my English.
